src/main/java/io/github/moulberry/notenoughupdates/profileviewer/SkyblockProfiles.java

```diff
@@ -484,6 +487,175 @@ public class SkyblockProfile {
 		private PlayerStats.Stats stats;
 		private Long networth = null;
 		private SoopyNetworth soopyNetworth = null;
+		private MuseumData museumData = null;
+		private final AtomicBoolean updatingMuseumData = new AtomicBoolean(false);
+
+		public class MuseumData {
+			private long museumValue;
+			private final Map<String, JsonArray> weaponItems = new HashMap<>();
+			private final Map<String, JsonArray> armorItems = new HashMap<>();
+			private final Map<String, JsonArray> raritiesItems = new HashMap<>();
+			private final List<JsonArray> specialItems = new ArrayList<>();
+			private final Map<String, Pair<Long, Boolean>> savedItems = new HashMap<>();
+
+			private MuseumData(JsonObject museumJson) {
+				JsonObject museum = Constants.MUSEUM;
+				if (museumJson == null || museumJson.isJsonNull() || museumJson.get("members").isJsonNull() || museum == null) {
+					museumValue = -2;
```

Comment:
I think its more helpful to show the repo notification here `Utils.showOutdatedRepoNotification()`